### ISSUE TITLE
Function even_parity is actually odd_parity

### ISSUE THREAD

**Issue by glinders** [smunaut/osmo-pocsag#1]
Function even_parity(uint32_t x) in file src/pocsag.c returns a '1' if parity is odd and a '0' if parity is even.

The name of the function suggests that it returns 'true' (i.e. '1') if parity is even.

Refer to attached source file:  [parity.zip](https://github.com/smunaut/osmo-pocsag/files/3283566/parity.zip)

Compile and run with: gcc parity.c -o parity && ./parity

**Comment by nospam2000** [comment]
it's a question of the point of view and the expectation:
 a) **check** if the parity is even and then return true
 b) **make** the sum of all all bits (including the returned one) even

so dependent if you want "check" or "make" the expectation is different

**Comment by glinders** [comment]
Yes, I agree but the name of your function doesn't give a clear hint as to what it does and there is no API description either.

I normally add an 'action verb' in the function name to make it clear; is_parity_even(val) or send_pocsag_message(msg).

**Comment by nospam2000** [comment]
I completely agree with you!
